### DISCUSSION TITLE
支持ES数据批量导入milvus的工具

### DISCUSSION THREAD

**Discussion by yangwentaoTi** [milvus-io/milvus#18001] [Ideas & Feature requests]
想要一个可以ES数据批量导入milvus的工具

**Comment by yhmo** [comment]
目前阶段我们先得把2.1弄稳定。如果ES数据能导出成特定json格式的话，milvus近期有个bulkload的功能可以通过json文件批量导入，不过得等2.1稳定后才好发布。
